modules/git/repo.go

```diff
@@ -147,8 +147,13 @@ func CloneWithArgs(ctx context.Context, from, to string, args []string, opts Clo
 		opts.Timeout = -1
 	}
 
-	_, err = cmd.RunTimeout(opts.Timeout)
-	return err
+	var stderr = new(bytes.Buffer)
+	err = cmd.RunWithContext(&RunContext{
+		Timeout: opts.Timeout,
+		Stdout:  new(bytes.Buffer),
```

Comment:
i thin there was a dummy reader for this ...

Can we make this field optional and check in RunWithContext if nil set dummy reader?

Reply:
I have removed this changes from this PR.